gwpy/cli/cliproduct.py

```diff
@@ -639,18 +639,20 @@ def set_legend(self):
         return leg
 
     def set_title(self, title):
-        """Set the title for this plot.
+        """Set the title(s) for this plot.
 
         The `Axes.title` actually serves at the sub-title for the plot,
         typically giving processing parameters and information.
         """
+
         if title is None:
-            title = self.get_title().rstrip(', ')
-        if self.usetex:
-            title = label_to_latex(title)
-        if title:
-            self.ax.set_title(title, fontsize=12)
-            self.log(3, ('Title is: %s' % title))
+            title = [self.get_title().rstrip(', ')]
+        for title_line in title:
```

Comment:
@areeda, if the user gives multiple arguments to `--title`, doesn't this mean that only the last given `title_line` will be preserved?

Reply:
@areeda, ping?